src/Microsoft.Extensions.PlatformAbstractions/Native/NativeMethods.Darwin.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Extensions.PlatformAbstractions.Native
+{
+    internal static partial class NativeMethods
+    {
+        public static class Darwin
+        {
+            private const int CTL_KERN = 1;
+            private const int KERN_OSRELEASE = 2;
+
+            public unsafe static string GetKernelRelease()
+            {
+                // This is way to small. Using it to test errno/GetLastError
```

Comment:
That comment shouldn't even be there. I put it there when I had set BUFFER_LENGTH to 2 in order to see if `sysctl` returned an error when the buffer was too small